packages/sanity/src/desk/panes/document/timeline/timelineMenu.tsx

```diff
@@ -45,12 +45,6 @@ export function TimelineMenu({chunk, mode, placement}: TimelineMenuProps) {
     setOpen(false)
   }, [setTimelineMode])
 
-  useEffect(() => {
-    if (open && isDeleted) {
-      handleClose()
-    }
-  }, [open, isDeleted, handleClose])
```

Comment:
**suggestion**: consider removing, as this can play havoc when using review changes on deleted documents (see below).